alchemy-web/src/pages/privacy.astro

```diff
@@ -0,0 +1,113 @@
+---
+/**
+ * Alchemy Privacy Policy
+ * Generated automatically from repository inspection of telemetry implementation.
+ * This page describes how the Alchemy CLI and related tools collect and use data.
+ */
+export const prerender = true; // generate at build time
+
+/**
+ * If your site has a global layout you can wrap this page.
+ * We keep this file standalone to avoid coupling to layouts that may not exist.
+ */
+---
+
+<style>
+  :root {
+    --content-max-width: 60ch;
+  }
+  main {
+    max-width: var(--content-max-width);
+    margin-inline: auto;
+    padding: 2rem 1rem;
+    line-height: 1.6;
+  }
+  h1, h2, h3 {
+    line-height: 1.25;
+    margin-top: 2rem;
+  }
+  h1 {
+    font-size: 2rem;
+    margin-top: 0;
+  }
+  ul {
+    padding-left: 1.25rem;
+  }
+</style>
+
+<main>
+  <h1>Privacy&nbsp;Policy</h1>
+  <p>Last updated: <!-- Will be substituted at build time -->{new Date().toLocaleDateString()}</p>
```

Comment:
This makes me a little nervous because it updates every build not on changes so users might think something changed when it didn't. I try and read privacy policies for SaaS stuff by looking at the data and skimming for what I need (e.g is my user info being sold)

I think it's better to have this be a static value that we update manually

Reply:
Good point. will change